examples/tensorflow/segmentation/configs/quantization/mask_rcnn_coco_int8.json

```diff
@@ -42,6 +42,7 @@
         "{re}.*tf.math.reduce_max.*",
         "{re}.*tf.math.reduce_min.*",
         "{re}.*tf.nn.avg_pool.*",
+	"{re}.*tf.linalg.matmul*",
```

Comment:
```suggestion
	    "{re}.*tf.linalg.matmul*",
```

Reply:
@dongkwan-kim01 Could you please apply this comment?